src/EmailParser.php

```diff
@@ -56,7 +56,7 @@ private function processLocalPart(): Result
         $localPartParser = new LocalPart($this->lexer);
         $localPartResult = $localPartParser->parse();
         $this->localPart = $localPartParser->localPart();
-        $this->warnings = array_merge($localPartParser->getWarnings(), $this->warnings);
+        $this->warnings = [...$localPartParser->getWarnings(), ...$this->warnings];
```

Comment:
Can you explain what is the benefit of this change?

Reply:
I'm not commenting on all the instances of similar changes to avoid generating noise.

Reply:
It's supposed to be a performance-related change. See [this benchmark](https://onlinephp.io/c/3038f).